proto/install.sh

```diff
@@ -0,0 +1,5 @@
+PB_REL="https://github.com/protocolbuffers/protobuf/releases"
+PB_VERION="3.13.0"
+curl -LO $PB_REL/download/v${PB_VERION}/protoc-${PB_VERION}-linux-x86_64.zip
+unzip protoc-${PB_VERION}-linux-x86_64.zip -d $HOME/.local
+export PATH="$PATH:$HOME/.local/bin"
```

Comment:
I installed protoc from the package manager, but this should ideally be part of the Makefile. However, this script will not be sufficient for everyone as it only works on one system/arch combination. I would rather exclude it for now and add a different installation method.

Reply:
Thanks. Addressed with a comment in the README instead